MyPerfectOnboardingApplication/MyPerfectOnboarding.Api.Tests/Utils/Extensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace MyPerfectOnboarding.Api.Tests.Utils
+{
+    public static class Extensions
```

Comment:
Extensions is generic name. What is the extension used for - that should be described in the name.